index.rst

```diff
@@ -352,8 +434,8 @@ For example, Google Filestore (useful for prototyping and test installations) su
 Other possible file systems (such as cluster file systems like GPFS or Lustre) are generally not available as standard services in cloud environments, which are used for prototyping and testing and which ideally should match the Data Facility environment.
 
 AFS and related technologies such as AuriStor deserve some separate discussion.
-AFS-based file systems are uniquely able to expose the same file system to the user's local machine and to the notebook aspect and internal Science Platform services.
-This neatly solves the problem of synchronizing files from a user's machine to their running notebook or their collaborators, which would be a significant benefit.
+AFS-based file systems are uniquely able to expose the same file system to the user's local machine and to the notebook aspect, portal aspect, and internal Science Platform services.
```

Comment:
This is not the only place it appears, but in general the project typography is to capitalize the Aspect names: "Notebook Aspect", "Portal Aspect", "[Web] API Aspect".